product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx

```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
 
 ## Installation on Kubernetes
 
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
+
+Export the appropriate value as an environment variable EDB_SUBSCRIPTION_PLAN.
+
+```shell
+export EDB_SUBSCRIPTION_PLAN=enterprise
+```
+
+then export the Repos 2.0 token to an environment variable EDB_SUBSCRIPTION_TOKEN.
+
+```shell
+export EDB_SUBSCRIPTION_TOKEN=<your-token>
+```
+
+!!! Warning
+The token is sensitive information. Please ensure that you don't expose it to unauthorized users.
+!!!
+
 ### Using the Helm Chart
 
 The operator can be installed using the provided [Helm chart](https://github.com/EnterpriseDB/edb-postgres-for-kubernetes-charts).
 
 ### Directly using the operator manifest
 
+#### Install the EDB pull secret
+
+Before installing EDB Postgres for Kubernetes, you need to create a pull secret for EDB software in the `postgresql-operator-system` namespace.
```

Comment:
I know you already mention "EDB software" in the section on getting the token, but I think this paragraph is clearer if saying the pull secret gives access to the private registries. I.e. not to "EDB software" in general.

Reply:
```suggestion
Before installing EDB Postgres for Kubernetes, you need to create a pull secret for EDB private registries in the `postgresql-operator-system` namespace.
```